src/list.js

```diff
@@ -0,0 +1,13 @@
+export function toggleCompleteTask(className) { /* eslint-disable-line no-unused-vars */
+  if (className.style.textDecoration === 'line-through') {
+    className.style.textDecoration = 'none';
+  } else {
+    className.style.textDecoration = 'line-through';
+  }
+}
+
+export function strikeText(checked, className) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.